docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -337,62 +364,64 @@ class Sidebar extends React.Component<any, any> {
       {
         key: 'chatMssages',
         title: { content: 'Chat Messages', as: NavLink, to: '/prototype-chat-messages' },
+        public: false,
```

Comment:
nit:
add `public: true` as required type for `prototypesTreeItems`. it will become:
```typescript
const prototypesTreeItems: (ShorthandValue & { key: string; public: boolean })[] = []
```